lib/get-page-data.js

```diff
@@ -6,16 +6,21 @@ function getPageData(html) {
     const $ = cheerio.load(html);
     const $title = $('h1').first();
     const $subtitle = $('p').first();
+    const $hr = $('hr').last();
+    const footerContent = $hr.nextAll();
+    const footer = $(footerContent).map((i, elem) => $(elem)).get().join('');
```

Comment:
`.map((i, elem) => $(elem).html()).join('')`? I would expect we needed to get the HTML somewhere.